src/Css/Style.php

```diff
@@ -2073,6 +2073,10 @@ protected function _set_style_side_type($style, $side, $type, $val)
 
                 $this->_props_computed[$prop] = $has_line_style ? $val_computed : 0;
             }
+        } elseif (($style === "border" || $style === "outline") && $type === "style") {
+            if (in_array($val, Style::$BORDER_STYLES)) {
+                $this->_props_computed[$prop] = $val;
+            }
```

Comment:
For consistency, would set `$this->_props_computed[$prop]` to `null` otherwise, as this might be called as part of one of the public setter methods.

Reply:
Good catch I missed that it wasn't being set otherwise.